docs/content/blog/authelia-traefik-setup-guide/index.md

```diff
@@ -166,30 +192,11 @@ tls:
         - TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
         - TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305
 ```
-Create acme.json with 600 permissions before starting Traefik: `touch acme.json && chmod 600 acme.json`
+Note: Create acme.json with 600 permissions before starting Traefik.
 
 ### Domain Configuration
-```yaml{title="traefik/config/static.yml"}
-http:
-  routers:
-    traefik:
-      rule: Host(`traefik.example.com`)
-      entrypoints:
-        https:
-      service: api@internal
-      middlewares:
-        - authelia
-      tls:
-        certResolver: cloudflare
-        domains:
-          ### Domain configuration
-          ### Add your domains here for TLS certificate generation
-          - main "example.com"
-            sans:
-              - "*.example.com" # Wildcard certificates require the DNS challenge.
-          #- main "example2.com" ## Add additional domains as needed.
-          #  sans:
-          #    - "*.example2.com"
+```yaml{title="traefik/config/static.yaml"}
+##This file can be used to define dynamic routers/services/middlewares.
```

Comment:
```suggestion
```yaml{title="traefik/config/dynamic.yaml"}
##This file can be used to define dynamic routers/services/middlewares.
```

Let's name this accordingly so it's less ambiguous.